src/collectives/device/common.h

```diff
@@ -449,7 +449,6 @@ struct RunWork {
     int wid = threadIdx.x / WARP_SIZE;
     ncclWorkElem* we = w->header.type == ncclWorkTypeRegColl ? &w->regElems[0].elem : &w->elems[0];
     int stride = w->header.type == ncclWorkTypeRegColl ? sizeof(ncclWorkElemReg) : sizeof(ncclWorkElem);
-    #pragma unroll 1
```

Comment:
Suggest to revert this change. Original unroll 1 is in NCCL. Changing this may have unexpected effect.